### ISSUE TITLE
Tabbed navigation interface lacks proper list semantics [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#230]
Without the use of list semantics, the tabbed navigation interface doesn't convey to TalkBack how many button tabs are present. The interface has 4 buttons "NY COVID Data", "My Health Log", "My COVID Alerts", and "App Settings", and should be announced as "NY COVID Data, tab, button, 1 of 4", "My Health Log, tab, button, 2 of 4", and so on.

![health intro - tabbed interface selected button](https://user-images.githubusercontent.com/7034772/92000677-c4c6af80-ed0b-11ea-9493-5690ffa48d4d.png)

Severity: High

**Remediation Guidance:**

- Set the `accessibilityRole` property on the container for the list of tabs, and set its value to `tablist`.
- Set the `accessibilityRole` property on each tab, and set its value to `tab`.
- Apply the `accessibilityState` object to each property, setting its type to `selected`, and then make sure it is set to `yes` for the (visually) selected tab in the list and `no` for all other tabs. Remember to programmatically update this object to match the visual state at all times.
- Apply a label to each tab to indicate where it falls in the order of the four (4) tabs, for example `accessibilityLabel="2 of 4"`

**Comment by helloantoine** [comment]
As well, we should callout the active tab: 
In our case: "My Health Log, **selected** tab, button, 2 of 4"

**Comment by floridemai** [comment]
@jonnyjames @helloantoine This has now been addressed. I did add a hint which I'm not sure is the right thing to do. We can remove that if not needed separately.

**Comment by jonnyjames** [comment]
@floridemai @helloantoine I can confirm this issue has been addressed. **Note:** for the hint, I don't think it is necessary, since the existing tab label text is fairly self-explanatory. I am closing this issue.
